Bixal/move.mil
# Bug fix tabbing in tutorial content types

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Remove focus (tabindex) from non-interactive elements.
- Makes sure tab/focus doesn't occur across slides (in page mode). Focus moves into and out of current slide only.
- Adds a screenreader-only button for "next" after the slide content.

## Testing

To verify the changes proposed in this pull request…

1. pull code locally
2. make cr
3. navigate /tutorials
4. tab through carousel

**@old behavior**

![tab-trap](https://user-images.githubusercontent.com/37077057/38952497-e62e2ae8-4319-11e8-82ed-397849ba1753.gif)

**@new behavior**

![no-tab-trap](https://user-images.githubusercontent.com/37077057/38952525-0040e132-431a-11e8-81d9-a5c75d639533.gif)

5. activate voice over (command+f5 on mac)
6. tab to through first slide/group in carousel, you should encounter a previous and a next button, but not see it visually.

**example:**

blackbox is focused on a next button

![screen shot 2018-04-18 at 3 07 41 pm](https://user-images.githubusercontent.com/37077057/38952619-4f30b7ea-431a-11e8-9512-b6a7da8e3bb3.png)
